dapp/src/helpers/claims.ts

```diff
@@ -0,0 +1,270 @@
+import SvelteComponentDev from '*.svelte';
+import {LinkInput, PersonOutlined, TwitterIcon} from 'components';
+import * as tzp from 'tzprofiles';
+
+
+const exhaustiveCheck = (arg: never) => {
+  // Forces the type checker to complain if you've missed a sum type.
+  // See https://dev.to/babak/exhaustive-type-checking-with-typescript-4l3f
+}
+
+// The types of claims supported by the UI.
+export type ClaimType = "basic" | "twitter"
+
+// TODO: Type better? Define what VCs look like generically?
```

Comment:
I guess we can put that in the bucket for a better WASM interface

Reply:
It would be nice if we could have a typescript interface representing a VC, but I don't know how practical that is.

Reply:
Yeah, not sure. The version made with neon translates the Rust struct but then I couldn't figure out how to use it :sweat_smile: